### DISCUSSION TITLE
Todavia hay problemas de la componente texto subtipo "long_text" con "short_text"

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#301] [4. Bug Reports] [answered]
Hola

Dejo los links de referencia:
https://github.com/Cotalker/documentation/discussions/126
https://github.com/Cotalker/documentation/discussions/199

Todavía se encuentra el problema, de aquellas componentes de tipo texto antiguas, que se crearon como "long_text", antes de dejar y validar sólo a "short_text". Como las discusiones son de Abril y Mayo pasado respectivamente, no sé si ya estaría en una próxima planificación.

Adjunto SS y codigo de referencia.

**En el survey builder**
![image](https://user-images.githubusercontent.com/61419471/123487459-f03cc280-d5db-11eb-9ee5-27b26c3c4b25.png)

**En base de datos**
```
{ 
    "_id" : ObjectId("60185c27fda78200088bc0be"), 
    "command" : {
        "commands" : [

        ], 
        "resetIdentifiers" : [

        ], 
        "isCommanded" : null, 
        "values" : [

        ]
    }, 
    "display" : [

    ], 
    "code" : [

    ], 
    "isActive" : true, 
    "isSystemModel" : false, 
    "contentType" : "application/vnd.cotalker.survey+textinput", 
    "help" : null, 
    "identifier" : "nombre_del_proyecto_colombia", 
    "isReadOnly" : false, 
    "max" : NumberInt(5000), 
    "min" : NumberInt(1), 
    "required" : true, 
    "subtype" : "long_text", 
    "company" : ObjectId("5eaaf0d4f999796a1563e002"), 
    "modifiedAt" : ISODate("2021-04-22T15:17:21.011+0000"), 
    "__v" : NumberInt(0)
}
```

**En el momento de guardar**
![image](https://user-images.githubusercontent.com/61419471/123487557-21b58e00-d5dc-11eb-92ed-0fb49d6f03a6.png)

**Qué desventaja tiene (para las antiguas implementaciones**
Que si se agregan nuevos campos, no se guarden porque puede existir un elemento que tenga ese problema y no se pueda guardar.

**Cómo se podría solucionar y posibilidad de un impacto**
1.- Se podría actualizar masivamente desde la base de datos las questions con query {"subtype" : "long_text"}, pero me dejaría la siguiente interrogante:
           - Existiría la posibilidad que genere algún error dentro del ambiente o del traspaso a otro ambiente de trabajo (prod a staging, staging a prod, dev a alguno de estos o local a alguno de éstos, etc?

2.- No se podría agregar la condicion que acepte long_text con short_text?

Gracias => 

PD: tengo malo las teclas: "Supr", "¿", "Impr Pant", si hay algo que no cuadra bien es por eso :(

**Comment by gmoplaza** [comment, marked as answer]
@NicolasPLG que raro que aún tengas problemas, lo voy a revisar ahora mismo y te cuento que descubro, disculpa y gracias por notificar

**Reply by gmoplaza** [reply]
@NicolasPLG se está subiendo el fix a producción

**Reply by NicolasPLG** [reply]
gracias @gmoplaza
